adapters/audienceNetwork/facebook.go

```diff
@@ -424,15 +424,15 @@ func Builder(bidderName openrtb_ext.BidderName, config config.Adapter, server co
 		return nil, errors.New("AppSecret is not configured. Did you set adapters.facebook.app_secret in the app config?")
 	}
 
-	bidder := &FacebookAdapter{
-		URI:        config.Endpoint,
+	bidder := &adapter{
+		uri:        config.Endpoint,
 		platformID: config.PlatformID,
 		appSecret:  config.AppSecret,
 	}
 	return bidder, nil
 }
 
-func (fa *FacebookAdapter) MakeTimeoutNotification(req *adapters.RequestData) (*adapters.RequestData, []error) {
+func (fa *adapter) MakeTimeoutNotification(req *adapters.RequestData) (*adapters.RequestData, []error) {
```

Comment:
I think this variable name should be `a` instead of `fa`?

Reply:
Done